issues/bridge/FACES-1470-portlet/src/main/java/com/liferay/faces/issues/bean/AS7LeakViewScopeInjectedBeanInnerClass.java

```diff
@@ -65,7 +65,7 @@ public void setFoo(String foo) {
 		this.foo = foo;
 	}
 
-	protected class AS7LeakAttribute extends NameValuePair<Object, Object> implements Serializable {
+	private static class AS7LeakAttribute extends NameValuePair<Object, Object> implements Serializable {
```

Comment:
@pcwhite Please delete AS7LeakViewScopeInjectedBeanInnerClass.java from the pull request, squash your commits, and push your branch again.